src/test/java/uk/gov/companieshouse/items/orders/api/interceptor/UserAuthorisationInterceptorTest.java

```diff
@@ -81,23 +81,40 @@ public void doesNotAuthoriseIfAuthenticatedUserDidNotCreateTheCertificateWhenReq
         when(request.getMethod()).thenReturn(HttpMethod.GET.toString());
         when(request.getAttribute(HandlerMapping.URI_TEMPLATE_VARIABLES_ATTRIBUTE)).thenReturn(map);
         when(request.getHeader(ERIC_IDENTITY_HEADER_NAME)).thenReturn(ERIC_IDENTITY_VALUE);
+        when(request.getHeader(ERIC_IDENTITY_TYPE_HEADER_NAME)).thenReturn(ERIC_IDENTITY_TYPE_OAUTH2_VALUE);
         when(service.getCertificateItemById(ITEM_ID)).thenReturn(Optional.of(item));
 
         assertFalse(userAuthorisationInterceptor.preHandle(request, response, null));
     }
 
     @Test
-    @DisplayName("Does not authorise if Certificate is not found when request method is GET")
-    public void willNotAuthoriseIfCertificateIsNotFound() {
+    @DisplayName("Does not authorise if Certificate is not found when request method is GET for a user")
+    public void willNotAuthoriseIfCertificateIsNotFoundAndOAuth2() {
         Map<String, String> map = new HashMap<>();
         map.put("id", ITEM_ID);
 
         when(request.getMethod()).thenReturn(HttpMethod.GET.toString());
         when(request.getAttribute(HandlerMapping.URI_TEMPLATE_VARIABLES_ATTRIBUTE)).thenReturn(map);
         when(request.getHeader(ERIC_IDENTITY_HEADER_NAME)).thenReturn(ERIC_IDENTITY_VALUE);
+        when(request.getHeader(ERIC_IDENTITY_TYPE_HEADER_NAME)).thenReturn(ERIC_IDENTITY_TYPE_OAUTH2_VALUE);
         when(service.getCertificateItemById(ITEM_ID)).thenReturn(Optional.empty());
 
         assertFalse(userAuthorisationInterceptor.preHandle(request, response, null));
     }
 
+    @Test
+    @DisplayName("Authorise if get and an API key is used")
+    public void willAuthoriseIfRequestIsGetAndAPIKey() {
+        when(request.getMethod()).thenReturn(HttpMethod.GET.toString());
+        when(request.getHeader(ERIC_IDENTITY_TYPE_HEADER_NAME)).thenReturn(ERIC_IDENTITY_TYPE_API_KEY_VALUE);
+        assertTrue(userAuthorisationInterceptor.preHandle(request, response, null));
+    }
+
+    @Test
+    @DisplayName("Authorise if get and an API key is used")
```

Comment:
"Will not authorise if post and an API key is used"?

Reply:
From the architecture diagram, api keys can be used to get certificates. So i've limited access to other request methods.